packages/lambda-xyz/src/cli/serve.ts

```diff
@@ -50,7 +50,7 @@ async function main(): Promise<void> {
             const data = await lambda.handleRequest(
                 {
                     httpMethod: 'get',
-                    path: `/v1/tiles/${z}/${x}/${y}.png`,
+                    path: `/v1/tiles/aerial/3857/${z}/${x}/${y}.png`,
```

Comment:
Would it not make sense to detect if webp is available in the browser (just about all now I think), then use that? It will really reduce the bandwidth.

Reply:
This is just a test server

Reply:
But yes for the html landing page we plans to serve webp